lighthouse-core/gather/gather-runner.js

```diff
@@ -585,6 +588,8 @@ class GatherRunner {
    * @param {LH.Gatherer.PassContext} passContext
    */
   static async populateBaseArtifacts(passContext) {
+    const status = {msg: 'Populate base artifacts', id: 'lh:gather:popBaseArtifacts'};
```

Comment:
```suggestion
    const status = {msg: 'Populate base artifacts', id: 'lh:gather:populateBaseArtifacts'};
```

does this hit 100? :)

Reply:
looks like avg monitor size grows ~3% a year so our line column should too https://techtalk.pcmatic.com/2017/01/16/pc-monitor-display-size/#:~:text=PC%20Monitor%20Display%20Sizes%20Continues%20to%20Grow&text=Our%20analysis%20shows%20that%20PC,comes%20in%20at%2016.1%20inches.

starting from 2015, that means 2021 should bring us to 119 characters. So 120 :)


















\s

Reply:
But really, for this, I'd prefer using no abbreviation ("pop" implies a stack...). you can put a newline just after the `=` to accommodate the linter.

Reply:
as it turns out..

```js
`    const status = {msg: 'Populate base artifacts', id: 'lh:gather:populateBaseArtifacts'};`.length === 91
```

so no need :)